app/__init__.py

```diff
@@ -81,13 +83,22 @@ def create_app(application):
     logging.init_app(application, statsd_client)
     firetext_client.init_app(application, statsd_client=statsd_client)
     mmg_client.init_app(application, statsd_client=statsd_client)
+
     aws_ses_client.init_app(application.config['AWS_REGION'], statsd_client=statsd_client)
+    aws_ses_stub_client.init_app(
+        application.config['AWS_REGION'],
+        statsd_client=statsd_client,
+        stub_url=os.environ.get('SES_STUB_URL')
+    )
+    # If a stub url is provided for SES, then use the stub client rather than the real SES boto client
+    email_clients = [aws_ses_client] if not os.environ.get('SES_STUB_URL') else [aws_ses_stub_client]
```

Comment:
Should we add something to ensure the stub is not used in production?

Reply:
Good call!

Reply:
I added a condition for that now 👍